splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/actions/GrantRevokeConstantOperation.java

```
@@ -55,23 +59,38 @@ public void executeConstantAction( Activation activation ) throws StandardExcept
         LanguageConnectionContext lcc = activation.getLanguageConnectionContext();
         TransactionController tc = lcc.getTransactionExecute();
 
-		List <PermissionsDescriptor> permissionsDescriptors = privileges.executeGrantRevoke( activation, grant, grantees);
+        Manager manager = EngineDriver.driver().manager();
+        if (privileges instanceof TablePrivilegeInfo) {
+            TablePrivilegeInfo info = (TablePrivilegeInfo) privileges;
+            if (!manager.isEnabled() && info.hasColumns()) {
+                throw StandardException.newException(SQLState.COLS_PERMS_EXCEPTION);
+            }
+        }
```

If the column is not present, you should throw a different exception